dockerfiles/startall.sh

```diff
@@ -142,16 +156,20 @@ if [[ $DEPLOY_ES == 1 ]] ; then
   echo -e "\n\nDeploying "myelastic4hpcclogs" - light-weight Elastic Stack:"
   if [[ -z "${DEP_UPDATE_ARG}" ]]; then
```

Comment:
minor: since the content of DEP_UPDATE_ARG is no longer releavant, would be best to change it to e.g. DEP_UPDATE=1